hail_search/search.py

```diff
@@ -0,0 +1,15 @@
+from hail_search.hail_search_query import QUERY_CLASS_MAP
+
+
+def search_hail_backend(request):
+    sample_data = request.pop('sample_data', {})
+
+    data_types = list(sample_data.keys())
+    single_data_type = data_types[0] if len(data_types) == 1 else None
+
+    sample_data = sample_data[single_data_type]
+    data_type = single_data_type
+    query_cls = QUERY_CLASS_MAP[single_data_type]
```

Comment:
It will be a key error when `single_data_type` is None. Should we report an error if the length of `data_types` is not 1?

Reply:
No, this is simply functionality thats not implemented yet but will be before we go live. It seemed unnecessary and too much work to add a bunch of error handling that we would immediately remove in a subsequent PR, when the end result would still be that search wouldn't work if you tried it. As I said in standup, this is only a partial implementation. The goal is to get stuff in in manageable chunks. If we are unwilling to merge code that does not have full functionality, we would have to review thousands of lines of code at once, which would result in worse review and worse code

Reply:
Can you add a comment about something to do before going live?

Reply:
I would rather not - there would be about 200 comments. Nothing works right now - none of the filters, multi family search, multi data type search, multi project search. The whole file would be comments. This is not a practical request

Reply:
if you are worried about missing stuff, at the very end I will have moved an entire branch over and I can check that there is nothing left on that branch thats different from dev, in order to confirm all fucntionality was transferred. But putting in hundereds of comments would be less accurate than that approach and a huge amount of my time